build_tools/third_party/nrfx/CMakeLists.txt

```diff
@@ -77,11 +77,12 @@ target_sources(nrfx_uart
 
 target_include_directories(nrfx_uart
   PUBLIC
+    ${nrfx_custom_DIR}
     ${nrfx_DIR}
     ${nrfx_DIR}/drivers/include/
     ${nrfx_DIR}/hal/
-    ${nrfx_DIR}/templates/
     ${nrfx_DIR}/mdk/
+    ${nrfx_DIR}/soc/
```

Comment:
This seems not strictly necessary as files from `${nrfx_custom_DIR}` where not included for this target till now. Anyway, I am generally fine with this change but I would like to have this in a separate commit to allow an easier tracking of changes as well as understanding the motivation behind.

Reply:
Good point. I would rather not include it. It will be an easy change if needed.